Polyhedron/demo/Polyhedron/Viewer.h

```diff
@@ -127,6 +127,10 @@ public Q_SLOTS:
   Viewer_impl* d;
   double prev_radius;
 
+public:
+  bool isRecent() const;
+  QOpenGLFunctions_4_3_Compatibility* recentFunctions();
```

Comment:
Thanks for the renaming of `isRecent()` to `isOpenGL_4_3()`.

My remark about the name of `recentFunctions()` remains: it must be renamed to something like `openGLFunctions_4_3`.